tests/sqllogictests/suites/base/11_data_type/11_0006_data_type_decimal

```diff
@@ -137,3 +137,30 @@ query I
 SELECT ANY(CAST(2.34 AS DECIMAL(6, 2)))
 ----
 2.34
+
+statement ok
+drop table if exists t
+
+statement ok
+drop table if exists t1
+
+statement ok
+create table t(c1 decimal(38,2))
+
+statement ok
+create table t1(c0 int, c1 decimal(39,2))
+
+statement ok
+select * from t order by c1
+
+## https://github.com/jorgecarleitao/parquet2/blob/main/src/schema/types/converted_type.rs#L25
+## https://github.com/jorgecarleitao/arrow2/blob/main/src/io/parquet/write/schema.rs#L323
+## the parquet2 and arrow2 not impl decimal256
```

Comment:
wait this done.

Reply:
Now decimal128 is already ok. So i think we can create a new issue to fix decimal256.